src/openag_brain/load_env_var_types.py

```diff
@@ -0,0 +1,37 @@
+class VariableInfo(dict):
```

Comment:
@rwdavis513 that makes sense. I appreciate that this lets us switch over without too many changes. We can always go back and remove the class later.